src/universal/modules/welcome/components/WelcomeWizardForms/Step1PreferredName.js

```diff
@@ -6,9 +6,25 @@ import WelcomeContent from '../WelcomeContent/WelcomeContent';
 import WelcomeHeader from '../WelcomeHeader/WelcomeHeader';
 import WelcomeHeading from '../WelcomeHeading/WelcomeHeading';
 import WelcomeLayout from '../WelcomeLayout/WelcomeLayout';
+import {cashay} from 'cashay';
+import {nextPage} from 'universal/modules/welcome/ducks/welcomeDuck';
 
 const Step1PreferredName = props => {
-  const {handleSubmit, preferredName} = props;
+  const {handleSubmit, preferredName, dispatch} = props;
+  const onPreferredNameSubmit = data => {
```

Comment:
Yeah, I like this better. It's easy for somebody to grok if they only had a single file to look at.